server/src/main/java/org/candlepin/resource/OwnerResource.java

```diff
@@ -894,12 +894,36 @@ public Owner updateOwner(@PathParam("owner_key") @Verify(Owner.class) String key
             }
         }
 
-        // Update the autobindDisabled field if the incoming value is null.
+        // Update the autobindDisabled field if the incoming value is not null.
         if (owner.getAutobindDisabled() != null) {
             toUpdate.setAutobindDisabled(owner.getAutobindDisabled());
         }
 
+        if (owner.getContentAccessMode() != null && config.getBoolean(ConfigProperties.STANDALONE)) {
```

Comment:
This is another part where the difference between null and an empty string will cause problems.